pkg/accounts/kubernetes/parse.go

```diff
@@ -38,10 +40,49 @@ func (k *AccountType) FromSpinnakerConfig(settings map[string]interface{}) (acco
 	} else {
 		return nil, fmt.Errorf("name is not a string")
 	}
+	auth, err := k.authFromSpinnakerConfig(a.Name, settings)
+	if err != nil {
+		return nil, err
+	}
+	a.Auth = auth
 	a.Settings = settings
 	return a, nil
 }
 
+func (k *AccountType) authFromSpinnakerConfig(name string, settings map[string]interface{}) (*v1alpha2.KubernetesAuth, error) {
+	kubeconfigFile, ok := settings["kubeconfigFile"]
```

Comment:
Can we use `inspect.GetObjectPropString(ctx, settings, "kubeconfigFile")`? It'd have the added benefit of supporting secrets.